composer.json

```diff
@@ -53,7 +53,7 @@
             "DDTrace\\Tests\\": "./tests/"
         }
     },
-    "minimum-stability": "dev",
+    "minimum-stability": "RC",
```

Comment:
no idea what this is. But if it's linked to the fact that otel was RC not so long ago, shall this change?

Reply:
You're right, it was being it was RC. Now that it isn't, it can be reverted :+1: